modules/kubernetes-addons/main.tf

```diff
@@ -147,6 +147,14 @@ module "cluster_autoscaler" {
   addon_context     = local.addon_context
 }
 
+module "coredns_autoscaler" {
```

Comment:
Why call it `coredns_autoscaler` instead of the generic `cluster_proportional_autoscaler`? I would imagine this we want customers to be able  re-purpose it for other applications?

Reply:
Yes, that's correct. The same helm addon can be used for other deployments as well. I have updated the docs with the details